build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
-    id "com.diffplug.spotless" version "5.12.1"
-    id "net.ltgt.errorprone" version "2.0.1"
+    id "com.diffplug.spotless" version "6.11.0"
```

Comment:
Why not 6.14.1?

Reply:
That's the same version that is being used in `zap-extensions`, can update in both (and everywhere).

Reply:
Done.